protocol/README.md

```diff
@@ -15,7 +15,7 @@ The canonical architecture contains four major components.
     * Return the resulting Claims Object in the signed IdP JWT.
   * A list of Certified OpenID Connect applications can be found at:  https://openid.net/developers/certified/
   * *Virtru Protocol Mapper* is our Keycloak-specific reference implementation of the above functionality.
-* *Attribute Provider* (AP) - A web service that will receive requests from our OIDC IdP custom code (ex: Virtru Protocol Mapper) containing information about authenticated subjet and return our custom OIDC/OAuth claims in response.  It is the responsibility of Attribute Provider to transform incoming 3rd party IdP claims/metadata to outgoing ABAC policy attributes.  It returns a TDF [Claims Object](../schema/ClaimsObject.md).
+* *Attribute Provider* (AP) - A web service that will receive requests from our OIDC IdP custom code (ex: Virtru Protocol Mapper) containing information about authenticated subjet and return our custom OIDC/OAuth claims in response.  It is the responsibility of Attribute Provider to transform incoming 3rd party IdP claims/metadata to a set of outgoing [Attribute Objects](../schema/AttributeObject.md).  It returns a TDF [Claims Object](../schema/ClaimsObject.md).
```

Comment:
I am fine with introverted attributes.

Reply:
👍 They're outgoing but introverted, as opposed to outgoing but extroverted.

Reply:
Nit: There are a couple grammar/spelling mistakes here.